builtin/providers/rancher/config.go

```diff
@@ -3,7 +3,7 @@ package rancher
 import (
 	"log"
 
-	rancherClient "github.com/rancher/go-rancher/client"
+	rancherClient "github.com/rancher/go-rancher/v2"
 	"github.com/raphink/go-rancher/catalog"
```

Comment:
The part that uses my fork of the lib for catalog templates. We could change that later though.